### PR TITLE
Updated conditions to retrieve primary keys in bulk_create() docs.

### PR DESCRIPTION
**Pull request by simkimsia**
… primary key

I rewrote the documentation as part of the feedback given in #17507

### CODE REVIEW

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2431,10 +2431,10 @@ This has a number of caveats though:
 * The model's ``save()`` method will not be called, and the ``pre_save`` and
   ``post_save`` signals will not be sent.
 * It does not work with child models in a multi-table inheritance scenario.
-* If the model's primary key is an :class:`~django.db.models.AutoField`, the
-  primary key attribute can only be retrieved on certain databases (currently
-  PostgreSQL, MariaDB, and SQLite 3.35+). On other databases, it will not be
-  set.
+* If the model's primary key is an :class:`~django.db.models.AutoField` and
+  ``ignore_on_conflict`` is False, the primary key attribute can only be
```
Comment (felixxm) [inline docs/ref/models/querysets.txt:2435]: We don't have `ignore_on_conflict` parameter.

Comment (felixxm) [inline docs/ref/models/querysets.txt:2435]:
```suggestion
  ``ignore_conflicts`` is False, the primary key attribute can only be
```

Comment (felixxm) [reply docs/ref/models/querysets.txt:2435]: @simkimsia If you don't have time, I can update it.

Comment (simkimsia) [reply docs/ref/models/querysets.txt:2435]: Yes please thank you 🙇🏻‍♂️

I'm away from keyboard

**Comment by felixxm** [comment]
@simkimsia Thanks :+1: Welcome aboard :boat:

**Pull request merged**